app/livechat/server/lib/Livechat.js

```diff
@@ -428,6 +428,8 @@ export const Livechat = {
 		Settings.findNotHiddenPublic([
 			'Livechat_title',
 			'Livechat_title_color',
+			'Livechat_show_message_character_limit',
+			'Livechat_message_character_limit',
```

Comment:
You need to retrieve here the `Message_MaxAllowedSize` setting as well.
```suggestion
			'Livechat_message_character_limit',
			'Message_MaxAllowedSize',
```